### PR TITLE
Allow cancel job on schema changes

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Partial solution for #2995

### CODE REVIEW

File: v2.1/cancel-job.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ summary: The CANCEL JOB statement stops long-running jobs.
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include [`IMPORT`](import.html) jobs and enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
+<span class="version-tag">New in v1.1:</span> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include schema changes, [`IMPORT`](import.html) jobs, and enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
```
Comment (mjibson) [inline v2.1/cancel-job.md]: The "new in v1.1" thing here is really weird. Although cancel came about in 1.1, it's no longer new. However the ability to cancel schema changes is new, while the rest aren't, except for IMPORT, which was added in 2.0. So LGTM I guess, but that's weird.

Comment (Amruta-Ranade) [reply v2.1/cancel-job.md]: I agree. I remember the Docs team had a discussion about this, but can't remember the decision. Will check with the team and update this doc accordingly.

Comment (Amruta-Ranade) [reply v2.1/cancel-job.md]: Done :)

File: v2.1/cancel-job.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ summary: The CANCEL JOB statement stops long-running jobs.
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include [`IMPORT`](import.html) jobs and enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
+The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running jobs, which include schema changes, [`IMPORT`](import.html) jobs, and enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) tasks.
```
Comment (jseldess) [inline v2.1/cancel-job.md]: We could tweak this a little to call out the newness:
> The `CANCEL JOB` [statement](sql-statements.html) lets you stop long-running tasks, including [`IMPORT`](import.html) jobs, enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) jobs, and as of v2.1, schema changes.

**Comment by Amruta-Ranade** [comment]
Merging this PR. Will open a follow-up PR if required.

**Pull request merged**